uber/templates/summary/csv_import.html

```diff
@@ -0,0 +1,45 @@
+{% extends "base-admin.html" %}
+{% block title %}CSV Import{% endblock %}
+{% block content %}
+
+<h2> CSV Table Imports </h2>
+This attempts to import a csv file for the table you select. <strong>Please do not refresh this page.</strong>
+
+<form action="import_model" method="post" enctype="multipart/form-data">
+    <select name="selected_model">
+        {% options tables %}
+    </select><br />
+    <input type="file" name="model_import" /><br />
+    <select name="date_format">
+        <option selected="selected" value="%Y-%m-%d">YYYY-MM-DD</option>
+        <option value="%Y/%m/%d">YYYY/MM/DD</option>
+        <option value="%m/%d/%Y">MM/DD/YYYY</option>
+        <option value="%d/%m/%Y">DD/MM/YYYY</option>
+    </select>
+    <input type="submit" value="Upload" />
+</form>
+<br/>
+{% if attendees %}
+        <div class="control-group"><h4>Imported Data</h4></div>
+    <table class="table table-striped datatable">
+    <thead><tr>
+        <th>ID</th>
```

Comment:
Yeah, you're correct that this would be annoying in Django (you'd need to write a custom filter) though it'll be easy once we migrate to Jinja.